index.bs

```diff
@@ -775,16 +816,17 @@ partial interface HTMLIFrameElement {
           <code>*</code></a>.
         1. Otherwise:
             1. Set |allowlist| to an new <a>ordered set</a>.
-            1. If |value| is the token `self`, append |origin| to |allowlist|.
+            1. If |value| is the token `self`, append the tuple |origin|, <code>False</code> to |allowlist|.
```

Comment:
You can use [tuple literal syntax](https://infra.spec.whatwg.org/#tuples) here -- just surround `origin, false` with parentheses